src/lib/cli/rules/templates/external-rule/templates/rule-doc.hbs

```diff
@@ -0,0 +1,44 @@
+# {{name}} (`{{normalizedName}}`)
+
+{{description}}
+
+## Why is this important?
+
+Explain why this {{#if multi}}package{{else}}rule{{/if}} is important for your users
+
+## What does the {{#if multi}}package{{else}}rule{{/if}} check?
+
+A bit more detail of what the {{#if multi}}package{{else}}rule{{/if}} does.
+
+{{#each rules}}
+{{#if ../multi}}
+## Rule: {{name}} (`{{normalizedName}}`)
+
+## What does the rule check?
+
+A bit more detail of what the rule does.
+
+{{/if}}
+### Examples that **trigger** the rule
+
+A list of code examples that will fail this rule.
+It's good to put some edge cases in here.
+
+### Examples that **pass** the rule
+
+A list of code examples that will pass this rule.
+It's good to put some edge cases in here.
+
+## Can the rule be configured?
+
+If this rule allows some configuration, please put the format and
+options for the user.
+
+## Further Reading
+
+What can the user read to know more about this subject?
+
+{{/each}}
+## License
+
+The code is available under the [Apache 2.0 license](LICENSE.txt).
```

Comment:
Since we don't provide this file, maybe this part should be removed?

Reply:
ok, it make sense